### DISCUSSION TITLE
Kiali Openid login issue

### DISCUSSION THREAD

**Discussion by subodh-net** [kiali/kiali#4152] [General]
![JWT 1](https://user-images.githubusercontent.com/61268073/124506087-8f399980-dde8-11eb-9e4d-79dacceea521.JPG)
![Kiali page 1](https://user-images.githubusercontent.com/61268073/124506089-906ac680-dde8-11eb-9950-b68708e6ab05.JPG)
![Kiali page 2](https://user-images.githubusercontent.com/61268073/124506091-91035d00-dde8-11eb-9fa0-d4df1685be7c.JPG)
![Kiali page 3](https://user-images.githubusercontent.com/61268073/124506094-91035d00-dde8-11eb-980c-e2a57f5ceaa5.JPG)
![Kiali page 4](https://user-images.githubusercontent.com/61268073/124506096-919bf380-dde8-11eb-96e1-1b3c45838ad8.JPG)
![Kiali page 5](https://user-images.githubusercontent.com/61268073/124506097-919bf380-dde8-11eb-8ae1-e61fe0b7a546.JPG)
![Kiali page 6](https://user-images.githubusercontent.com/61268073/124506099-92348a00-dde8-11eb-92a4-6c1dc57cffc2.JPG)
Hello Edgar

Q> Please, tell what Kiali version you are using.

Kiali UI
1.34.1 (cda505c688a0e8b6ce65dd80ca0203a5153d977d)
Kiali Server
v1.34.1 (31fc3596219485ab2e37d3968dc54910f3378992)
Kiali Container
v1.34.1

**Comment by israel-hdez** [comment]
@subodh-net Can you provide the logs of the Kiali pod generated after the failed authentication?
